includes/discovery/sensors/gw-eydfa.inc.php

```diff
@@ -0,0 +1,163 @@
+<?php
```

Comment:
I've had a quick general look at the MIB for this and it looks like this could be moved to yaml. Have you tried that?

Reply:
There are two points.
The table contains different sensor types and I dont know how to difference them.
The other is limits being this kind:
```
analogAlarmLO.13.1.3.6.1.4.1.17409.1.11.4.1.2.2
```
And yaml parser interpretes .13.1.3.6.1.4.1.17409.1.11.4.1.2.2 as index

Reply:
It seems like that is correct.  The MIB says the index is analogParameterOID.

13 is the length of the string, the rest is the analogParameterOID value.

Reply:
Yes, I know it is the correct Index. Now the problem is:

Those are the two state sensors oids, one for each power supply:
```
.1.3.6.1.4.1.17409.1.1.2.1.4.11.1.3.6.1.4.1.17409.1.11.100.0.2
.1.3.6.1.4.1.17409.1.1.2.1.4.11.1.3.6.1.4.1.17409.1.11.101.0.2
```
which translates to:
```
NSCRTV-HFCEMS-PROPERTY-MIB::discreteAlarmState.11.1.3.6.1.4.1.17409.1.11.100.0.2
NSCRTV-HFCEMS-PROPERTY-MIB::discreteAlarmState.11.1.3.6.1.4.1.17409.1.11.101.0.2
```
I did several tests (maybe all wrong) and I cant get the 2 sensors working.

In the other part of the yaml. I dont know how to walk the table which have several sensor types.
oaPumpTable have:
```
Current in mA (divisor 1000)
Current in A (divisor 100)
Temperature in C (divisor 10)
```
Also, the Hi/Lo limits also comes like the state sensors, with:
```analogAlarmLO/analogAlarmLOLO/analogAlarmHI/analogAlarmHIHI" . length . sensor_oid```

I think all this cant be achieved with yaml, but as I said I can be wrong.

Im open to ideas ;)

Reply:
Looks like you are right :( I've took a look at the data and without some work in the yaml processing code this isn't natively supported.